app/middleware/errors/not-found.error.js

```diff
@@ -0,0 +1,4 @@
+class NotFoundError extends Error {
```

Comment:
will rebase and replace with class from https://github.com/alphagov/pay-selfservice/pull/2397